glib-macros/tests/properties.rs

```diff
@@ -160,6 +160,10 @@ mod foo {
             weak_ref_prop: glib::WeakRef<glib::Object>,
             #[property(get, set)]
             send_weak_ref_prop: glib::SendWeakRef<glib::Object>,
+            #[property(get, default_value = 0, construct_only)]
```

Comment:
How would a default value work with construct only ?

Reply:
default actually *only* works for construct and construct_only